.github/workflows/update-cli-reference.yml

```diff
@@ -0,0 +1,46 @@
+on:
+  workflow_dispatch
+    
+jobs:
+  update-cli-reference:
+    name: Update the CLI reference
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - name: Download and prepare the am binary
+        run: |
+          mkdir /tmp
+          cd /tmp/
+          curl -L https://github.com/autometrics-dev/am/releases/latest/download/am-linux-x86_64 -o am
+          chmod u+x am
+
+      - name: Generate the CLI reference
+        run: |
+          /tmp/am markdown-help > src/pages/local-development/cli-reference.md
+          echo "am_version=$(/tmp/am --version | awk '{print $2}')" >> "$GITHUB_ENV"
+
+      - name: Cleanup
+        run: rm -r /tmp
+
+      - name: Create a branch and add the changed commit
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+          echo "branch_name=bot-cli-reference-$(date +'%Y-%m-%d')" >> "$GITHUB_ENV"
+          git add src/pages/local-development/cli-reference.md
+          git commit -m "Automated CLI reference update: $(date +'%Y-%m-%d')"
+          git push
```

Comment:
Shouldn't we first need to checkout the new branch defined on line 31?

Reply:
So the way I understood that PR action works is that it will create the branch itself, diff the changes with checked out `base` and move the changes there: https://github.com/marketplace/actions/create-pull-request#action-behaviour